CefSharp.BrowserSubprocess.Core/Async/JavascriptAsyncMethodWrapper.h

```diff
@@ -0,0 +1,35 @@
+// Copyright � 2010-2015 The CefSharp Project. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+#pragma once
+
+#include "JavascriptAsyncMethodHandler.h"
+#include "JavascriptCallbackRegistry.h"
+
+namespace CefSharp
+{
+    ref class CefBrowserWrapper;
```

Comment:
Is this required? I don't see any reference to `CefBrowserWrapper` in this class